backend/webdav/webdav.go

```diff
@@ -67,14 +69,17 @@ func init() {
 				Help:  "Owncloud",
 			}, {
 				Value: "sharepoint",
-				Help:  "Sharepoint",
+				Help:  "Sharepoint Online, authenticated by Microsoft OneDrive account.",
+			}, {
+				Value: "sharepoint-ntlm",
+				Help:  "Sharepoint with NTLM authentication. Usually self-hosted or company instances.",
```

Comment:
The most popular keyword for those SP installations seems to be "On-Premises".

Reply:
ok. will add a fix commit.

Reply:
fixed by commit https://github.com/rclone/rclone/pull/2921/commits/841bf20207699f3a35a0b009b549e33da40277cc